x/house/keeper/deposit.go

```diff
@@ -31,9 +34,9 @@ func (k Keeper) GetDeposit(ctx sdk.Context, depositorAddress, marketUID string,
 	return val, true
 }
 
-// GetAllDeposits returns all deposits used during genesis dump.
-func (k Keeper) GetAllDeposits(ctx sdk.Context) (list []types.Deposit, err error) {
-	store := k.getDepositsStore(ctx)
+// GetDeposits returns returns list of deposits.
```

Comment:
GetDeposits returns returns list of deposits. -> GetDeposits returns list of deposits.

Reply:
Resolved